wasmsdk/blobber.go

```diff
@@ -46,8 +46,12 @@ func createDir(allocationID, remotePath string) error {
 	return allocationObj.CreateDir(remotePath)
 }
 
-// lisBlobbersForFile returns details about
-func getFileStats(allocationID, remotePath string) ([]sdk.FileStats, error) {
+type FilesDetails struct {
+	RemotePath string `json:"remote_path"`
+}
```

Comment:
as this struct is no longer being used. can this be removed?

Reply:
@cnlangzi can you help to remove the unused `FilesDetails`

Reply:
@peterlimg  done